docsets/Vulkan/docset.json

```diff
@@ -1,10 +1,20 @@
 {
     "name": "Vulkan",
-    "version": "1.0.38",
+    "version": "1.0.39",
     "archive": "Vulkan.tgz",
     "author": {
         "name": "Lei Zhang",
         "link": "https://github.com/antiagainst"
     },
-    "aliases": ["vk", "GLNext", "Graphics API"]
+    "aliases": ["vk", "GLNext", "Graphics API"],
+    "specific_versions": [
+        {
+            "version": "1.0.39",
+            "archive": "versions/1.0.39/Vulkan.tgz"
+        },
+        {
+            "version": "1.0.38",
+            "archive": "versions/1.0.38/Vulkan.tgz"
```

Comment:
> And just to clarify, in the future, say if I want to update 1.0.100, I need to check in all the tarballs for 1.0.38 way up to 1.0.100?

Nope, no need for that. You just need to give the tarball once to the `specific_version`. Think of the stuff in `specific_versions` as completely separate from the main docset archive (i.e. the one at `docsets/Vulkan/Vulkan.tgz`).